nat-lab/tests/test_lana.py

```diff
@@ -715,10 +1117,42 @@ async def test_lana_with_disconnected_node() -> None:
 
 @pytest.mark.moose
 @pytest.mark.asyncio
-async def test_lana_with_second_node_joining_later_meshnet_id_can_change() -> None:
+@pytest.mark.parametrize(
+    "alpha_ip_stack,beta_ip_stack",
+    [
+        pytest.param(
+            IPStack.IPv4,
+            IPStack.IPv4,
+            marks=pytest.mark.ipv4,
+        ),
+        pytest.param(
+            IPStack.IPv6,
+            IPStack.IPv6,
+            marks=pytest.mark.ipv6,
+        ),
+        pytest.param(
+            IPStack.IPv4v6,
+            IPStack.IPv4v6,
+            marks=pytest.mark.ipv4v6,
+        ),
+        pytest.param(
+            IPStack.IPv4,
+            IPStack.IPv4v6,
+            marks=pytest.mark.ipv4v6,
+        ),
+        pytest.param(
+            IPStack.IPv6,
+            IPStack.IPv4v6,
+            marks=pytest.mark.ipv4v6,
+        ),
+    ],
+)
```

Comment:
Create a variable for this to remove repeating code.

Reply:
Done!